_attributes/common-attributes.adoc

```diff
@@ -146,6 +146,9 @@ endif::[]
 :SMProductVersion1x: 1.1.18.2
 //Windows containers
 :productwinc: Red Hat OpenShift support for Windows Containers
+// IBM Cloud
+:ibmcloudBMProductName: IBM Cloud Bare Metal (Classic)
+:ibmcloudBMRegProductName: IBM Cloud&#174; Bare Metal (Classic)
```

Comment:
Just curious -- are we required to include the registered trademark in the first instance of "IBM Cloud"? I hadn't seen that before in Red Hat documentation, so I'm just asking for my own knowledge.

Reply:
Hi @lahinson . Timing could not have been better: see, https://github.com/redhat-documentation/supplementary-style-guide/pull/359#discussion_r1265446103